RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -364,9 +364,11 @@ protected void executeRun(ExecutionContext context, boolean preventLoggingOnQuer
         String datasetID = context.getDataSet().getId();
         Exception eCopy = null;
         try {
+
+            context.analysis.getWorkflow().setupExecution(context);
```

Comment:
There's also finalize execution. I think I have a branch in my private repo where I also added the calls